heroic-component/src/main/java/com/spotify/heroic/aggregation/ChainInstance.java

```diff
@@ -202,6 +206,37 @@ public static AggregationInstance fromList(final List<AggregationInstance> chain
         return child.build();
     }
 
+    public List<AggregationInstance> getChain() {
+        return this.chain;
+    }
+
+    public boolean equals(final Object o) {
```

Comment:
It can. Its just a long, complicated class and only sort-of a data class - only one instance variable and not even a getter for that. I thought the effort of converting it wasn't worth it for just the equals/hashTo functions considering the size.